community/tutorials.rst

```diff
@@ -50,7 +50,7 @@ Text tutorials
 
 - `FinepointCGI website by Mitch <http://finepointcgi.io/>`__
 - `GDScript website by Andrew Wilkes <https://gdscript.com>`__
-- `Godot Recipes by KidsCanCode <http://godotrecipes.com/>`__
+- `Godot Recipes by KidsCanCode <https://kidscancode.org/godot_recipes/>`__
```

Comment:
```suggestion
- `Godot Recipes by KidsCanCode <https://kidscancode.org/godot_recipes/4.x/>`__
```

Given that this PR is for the master branch change it to this URL, not using the `4.x` makes the link default to 3.x recipes

Reply:
Done👍